src/sec_certs/dataset/cve.py

```diff
@@ -55,9 +55,15 @@ def __len__(self) -> int:
     def __eq__(self, other: object):
         return isinstance(other, CVEDataset) and self.cves == other.cves
 
+    def _filter_cves_with_cpe_configurations(self) -> None:
+        """
+        Method filters the subset of CVEs, which contain at least one CPE configuration.
+        """
+        self.cves_with_vulnerable_configurations = [cve for cve in self if cve.vulnerable_cpe_configurations]
```

Comment:
This should be set to empty list in `__init__`, so that we're explicit about its existance.